arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -232,5 +232,79 @@ public async Task PostGraphAsync_ShouldThrow_WhenGraphNameIsInvalid()
             Assert.Equal(HttpStatusCode.BadRequest, ex.ApiError.Code);
             Assert.Equal(1221, ex.ApiError.ErrorNum); // ARANGO_DOCUMENT_KEY_BAD
         }
+
+        [Fact]
+        public async Task DeleteEdgeDefinitionAsync_ShouldSucceed()
+        {
+            string edgeClx = nameof(DeleteEdgeDefinitionAsync_ShouldSucceed) + "_EdgeClx";
+
+            var createClxResponse = await _fixture.ArangoDBClient.Collection.PostCollectionAsync(
+                new PostCollectionBody()
+                {
+                    Name = edgeClx,
+                    Type = 3
+                });
+
+            Assert.Equal(edgeClx, createClxResponse.Name);
+
+            string graphName = nameof(DeleteEdgeDefinitionAsync_ShouldSucceed);
+
+            PostGraphResponse createGraphResponse = await _client.PostGraphAsync(new PostGraphBody()
+            {
+                Name = graphName,
+                EdgeDefinitions = new List<EdgeDefinition>()
+                {
+                    new EdgeDefinition()
+                    {
+                        Collection = edgeClx,
+                        From = new string[] { "FromCollection" },
+                        To = new string[] { "ToCollection" }
+                    }
+                }
+            });
+
+            Assert.Equal(HttpStatusCode.Accepted, createGraphResponse.Code);
+
+            var response = await _client.DeleteEdgeDefinitionAsync(graphName, edgeClx, new DeleteEdgeDefinitionQuery
+            {
+                WaitForSync = false,
+                DropCollections = true
+            });
+
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.Empty(response.Graph.EdgeDefinitions);
```

Comment:
I think we should also check that the collection has been dropped otherwise we don't know whether the drop collection query parameter worked properly.
Or am I thinking to much?